hub-js/src/local/storage/service.ts

```diff
@@ -182,4 +361,19 @@ export default class DelegatedStorageService {
 
     return this.registeredClients.get(id);
   }
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  private static convertToJSONIfObject(val: any) {
+    if (val instanceof Array || val instanceof Object) {
+      return JSON.stringify(val);
+    }
+    return val;
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  private encode(val: any) {
```

Comment:
Empty context results in internal server error:

query:

```graphql
mutation CreateTypeInstances {
  createTypeInstances(
    in: {
     typeInstances: [
      {
        alias: "helm-release"
        typeRef: { path: "cap.type.helm.chart.release", revision: "0.1.0" }
        value: {
          key: "test" # same as it was
        }
        backend: {
          id: "1cd21f38-49f0-42d9-b2f5-c3e69e00c896"
          context: {
            
          }
        }
      }  
      # {
      #   alias: "helm-release2"
      #   typeRef: { path: "cap.type.helm.chart.release", revision: "0.1.0" }
      #   value: {
      #     key: "test" # same as it was
      #   }
      #   # backend: {
      #   #   id: "123" # it's like that already.
      #   #   context: { # new property of type `Any!`
      #   #     name: "release-name",
      #   #     namespace: "release-namespace",
      #   #   }
      #   # }
      # }    
    ]
    usesRelations: [
      # {from: "helm-release", to: "helm-release2"}
      # {from: "54f39b4d-280b-406f-afa9-5645328fae70", to: "helm-release2"}
      # {from: "61d48278-3c5c-4eb5-8b6b-de4ce4582128", to: "helm-release2"}
    ]
  }
  ) {
    id
    alias
  }
}
```

output:

```json
{
  "errors": [
    {
      "message": "Cannot convert object to primitive value",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "createTypeInstances"
      ],
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR",
        "exception": {
          "stacktrace": [
            "TypeError: Cannot convert object to primitive value",
            "    at TextEncoder.encode (node:internal/encoding:333:32)",
            "    at DelegatedStorageService.encode (/Users/pkosiec/repositories/capact/hub-js/dist/local/storage/service.js:261:34)",
            "    at DelegatedStorageService.Delete (/Users/pkosiec/repositories/capact/hub-js/dist/local/storage/service.js:147:31)",
            "    at processTicksAndRejections (node:internal/process/task_queues:96:5)",
            "    at async createTypeInstances (/Users/pkosiec/repositories/capact/hub-js/dist/local/mutation/create-type-instances.js:44:9)"
          ]
        }
      }
    }
  ],
  "data": null
}
```

Reply:
Nope, it's not a problem with empty context. It was a problem with:
```
[Object: null prototype] { key: 'test' }
```
In general, we allow passing in `value` field JSON encoded value and also JS object. In case of our Go client, we marshal a given obj. In case of "manual" query you used the second option. As this was Obj with null prototype it was returning false for `val instanceof Object`. I adjusted our code to catch that.